carbon/app/[locale]/off-chain-vs-on-chain/page.tsx

```diff
@@ -15,8 +15,8 @@ export default function OffVsOnChainPage() {
       <PageHeader title={t`Off vs On-chain carbon`} />
       <div className={layout.cardStackedRows}>
         <div className={`${layout.cardRow} ${layout.prioritizeFirstCard}`}>
-          <VerraCreditsBreakdownCard />
-          <TokenizedCreditsByBridgeCard />
+          <VerraCreditsBreakdownCard className={layout.zIndexSeven} />
+          <TokenizedCreditsByBridgeCard className={layout.zIndexSix} />
```

Comment:
What does the z-index fixes?

Reply:
it fixes the tooltips being shown behind oder cards:

<img width="527" alt="grafik" src="https://github.com/KlimaDAO/klimadao/assets/138768831/70633a9a-c9e2-43b7-afa3-96b5c80f2a80">